### PR TITLE
[SCS] new recipe

### PR DESCRIPTION
**Pull request by ViralBShah**
Based on https://github.com/JuliaOpt/SCSBuilder/blob/master/build_tarballs.jl

### CODE REVIEW

**Comment by ViralBShah** [comment]
Requesting for a quick review before merging.

File: S/SCS/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+make BLASLDFLAGS="${blasldflags}" ${flags} out/libscsdir.${dlext}
+make BLASLDFLAGS="${blasldflags}" ${flags} out/libscsindir.${dlext}
+
+if [[ ${target} == *mingw* ]]; then
+    mkdir -p ${prefix}/bin
+    cp out/libscs*.dll ${prefix}/bin
+else
+    mkdir -p ${prefix}/lib
+    cp out/libscs*.${dlext} ${prefix}/lib
+fi
```
Comment (giordano) [inline S/SCS/build_tarballs.jl]: This is equivalent to
```suggestion
mkdir -p ${libdir}
cp out/libscs*.${dlext} ${libdir}
```

File: S/SCS/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libscsindir", :indirect),
+    LibraryProduct("libscsdir", :direct)
```
Comment (giordano) [inline S/SCS/build_tarballs.jl]: The names of the variables are common names, maybe better use the same name as the lib, as usual?
```suggestion
    LibraryProduct("libscsindir", :libscsindir),
    LibraryProduct("libscsdir", :libscsdir)
```

**Comment by ViralBShah** [comment]
We have done none of the name mangling for blas and 64-bit blas stuff. So perhaps need to dig deeper.

**Comment by ViralBShah** [comment]
Love it that they support all the options needed for 64-bit blas and name mangling.

**Pull request merged**